packages/styles/src/components/header/index.scss

```diff
@@ -4,12 +4,6 @@
 
 @use 'mixins' as *;
 
-@include media.min(lg) {
-  body:has(post-header) {
-    overflow-x: hidden;
-  }
-}
-
```

Comment:
This is needed for the main navigation overflow: ![image](https://github.com/user-attachments/assets/0166e610-df61-4e2c-9fc4-ecb60e64c50f)

As Cargo does not use this feature you may want to disable the tests for the moment.